juriscraper/opinions/united_states/state/neb.py

```diff
@@ -11,33 +11,29 @@ def __init__(self, *args, **kwargs):
         self.status = "Published"
 
     def _process_html(self):
-        path = {
-            "group": '//div[contains(@class, "view-grouping")]',
-            "date": './/span[contains(@class, "date-display-single")]',
-            "row": ".//tbody/tr",
-        }
-        for group in self.html.xpath(path["group"]):
-            date_element = group.xpath(path["date"])
-            if not date_element:
+        for table in self.html.xpath(".//table"):
+            date_tags = table.xpath("preceding::time[1]/text()")
+            if not date_tags:
+                print("NO DATE TAGS")
                 continue
-            date = date_element[0].text_content()
-            for row in group.xpath(path["row"]):
-                url = row.xpath(".//td[3]//a/@href")
-
-                # skip rows without url in third cell
-                if not url:
+            for row in table.xpath(".//tr"):
+                cells = row.xpath(".//td")
+                # make sure its a valid row - and not the ending of the table
+                if len(cells) != 3 or cells[0].xpath(".//text()") is None:
                     continue
 
-                docket = row.xpath(".//td[1]")[0].text_content().strip()
-                if docket == "A-XX-XXXX":
-                    continue
```

Comment:
Shouldn't we keep this? The rows on nebctapp that have this docket number most of the time don't have important content, seems noisy to add them to the DB as opinions

![image](https://github.com/freelawproject/juriscraper/assets/14970769/c50075b4-4a3b-40e2-883c-36762854fdf6)
![image](https://github.com/freelawproject/juriscraper/assets/14970769/59a7dcb2-c346-467f-9026-17dbaabd66e0)

Reply:
thanks for catching this